jenkins/build.jpl

```diff
@@ -115,9 +118,22 @@ node("builder") {
         }
     }
 
-    stage("Build") {
-        timeout(time: 180, unit: 'MINUTES') {
-            buildConfig(params.DEFCONFIG, kdir, kci_build)
+    stage("Delay") {
+        if (env.DELAY) {
+            echo "Builder has DELAY env set, sleeping for random seconds up to: " + env.DELAY
+            Random rand = new Random()
+            rdelay = rand.nextInt(env.DELAY.toInteger() + 1)
+            sleep(time: rdelay, unit: 'SECONDS')
+        }
+    }
+
+    def docker_image = "${params.DOCKERBASE}${params.COMPILER}_${params.ARCH}"
```

Comment:
This is the bit I think could be shown in the summary when the job starts.

Reply:
Ack